### DISCUSSION TITLE
GC - PS4 controller pairing issue

### DISCUSSION THREAD

**Discussion by Parrrrryma** [darthcloud/BlueRetro#541] [Q&A]
I am having issues with PS4 controllers pairing to my GC adapter. It will pair for a fraction of a second, then unpair. 

Is there anything settings wise to look into to resolve this?

1.8.1 HW2

**Comment by darthcloud** [comment]
PS4 usually work very well, if you got a PS4 maybe try to update the controller Firmware?

**Comment by RosenNX6** [comment]
I had the same issue and resolved it by resetting the controller. 
There is a small hole in the back of the PS4 controller where the reset button is located. Make sure that the controller is sufficiently charged, turned off, and unplugged from the usb and then use a paperclip to press that button for at least 5 seconds. Turn the controller back on and then it should work.
